docs/testing/config.md

```diff
@@ -1,160 +0,0 @@
----
```

Comment:
For each of the pages we moved/deleted, we'll need to set up redirects in the repo's `vercel.json` file. LMK if you need any help with that!

Reply:
No page urls were moved, all still point to the same content / all content remains on the same urls.

Do we want to update the urls and set up redirects instead?

Reply:
Oh, you're right - that's my fault. I looked at the overview page of the Stencil test runner and assumed that the other pages had changed as well. In that case, I'm OK with leaving them as is - sorry about that!

Reply:
np!

Reply:
this might be a pain - but if `docs/testing/stencil-testrunner/02-config/md` is the same file that used to be at `docs/testing/config.md` would it be possible to keep our git history around by moving the file using `git mv`?